var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -19,6 +19,9 @@ class Paraview(CMakePackage, CudaPackage):
 
     maintainers = ['chuckatkins', 'danlipsa']
 
+    # Release Candidate versions
+    version('5.9.0-RC2', sha256='a3e6f7db7804f11351df48781e8d67ff3e9c18a691bb9ef9a5485aa6520e70bc')
+
     version('develop', branch='master', submodules=True)
```

Comment:
We should call this master for consistency

Reply:
Changed in the last push. 

Thanks for the review!